payments/payment-provider.go

```diff
@@ -0,0 +1,99 @@
+package payments
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/statechannels/go-nitro/channel/state"
+)
+
+const alice = 0
+
+type (
+	// A PaymentProvider can be used to make a payment for a given channel.
+	PaymentProvider struct {
```

Comment:
Maybe we could use `Manager` or something instead of `Provider`, which is already in use in our lexicon.

Reply:
52825bce9952ba352b854e3b35e7cfbf4765c389

Reply:
52825bce9952ba352b854e3b35e7cfbf4765c389